.stylelintrc

```diff
@@ -1,9 +1,10 @@
 {
   "extends": [
-    "stylelint-config-standard",
+    "stylelint-config-standard"
```

Comment:
Personally I'm a fan of trailing commas because they always result in smaller diffs when you add/remove lines. Not sure if they're actually valid in JS though.

Reply:
I added the commas as they were before